### DISCUSSION TITLE
Extract .exe file

### DISCUSSION THREAD

**Discussion by KaliKody** [aonez/Keka#1437] [Q&A] [answered]
I read that you can use Keka to extract .exe files because I was trying to get Fallout 2 to work on my mac. I can't figure out how to get it to work because everytime I try it says "Error code 1 using "unar"" Unknown error. I can't find any posts about this.

**Comment by gingerbeardman** [comment, marked as answer]
I think a couple of crossed wires here:

- some .exe can be extracted if they are installers of certain types
- extracting an .exe is unlikely to get a PC game to work on your macOS

Can you tell us more about what your goal is? Particularly how extracting the .exe might help.

ps: I use [Whisky](https://github.com/Whisky-App/Whisky) to play PC games on my Apple Silicon M1 Pro Mac. There are other methods for older Macs.
